frontend/src/store.ts

```diff
@@ -19,6 +20,7 @@ export const store = configureStore({
     jiraVerifyResponse: jiraVerifyResponseReducer,
     pipelineToolVerifyResponse: pipelineToolResponseReducer,
     sourceControlVerifyResponse: sourceControlVerifyResponseReducer,
+    pipelineMetricsSettings: pipelineMetricsSettingsReducer,
```

Comment:
Put it as a child of metrics reducer, ref to https://github.com/au-heartbeat/HeartBeat/blob/main/frontend/src/context/config/configSlice.tsx#L62